src/containers/IbcRelayers/data.js

```diff
@@ -0,0 +1,25 @@
+export const ibcRelayers = [
+  {
+    key: "1",
+    relayer: "AUDIT.One",
+    email: "hello@audit.one",
```

Comment:
can you verify the contents here?

Reply:
This data is taken from figma. Can you check and provide me with correct details.